src/charactersheet/viewmodels/character/root/index.html

```diff
@@ -1,53 +1,71 @@
 <ul class="nav nav-tabs tabs">
-  <li role="presentation" data-bind="click: activateStatsTab, css: statsTabStatus">
+  <li role="presentation"
+      id="tabStatsLink"
```

Comment:
Is this the correct place to add the ID? Or should it be on the `<a>` tag?

Reply:
I believe so, it is functional in the test. I think it works because of the knockout binding.

Reply:
Cool